Microsoft.Toolkit.Uwp.UI/Extensions/ConditionalToolTip.cs

```diff
@@ -0,0 +1,76 @@
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
```

Comment:
We started flattening our namespace, so all extensions moved to just be in the `UI` domain.